packages/extract-react-types/src/index.js

```diff
@@ -664,12 +660,12 @@ converters.ObjectTypeAnnotation = (path, context): K.Obj => {
 };
 
 converters.ObjectTypeProperty = (path, context): K.Property => {
```

Comment:
any convention around immediate return arrows? eg.

`const x = b => ({
  thing: b.thing,
  ...other,
})`

Reply:
Yeah based on the rest of this file, it looks like it's been avoided, but i agree, much prefer this syntax. 

The `return` triggers my ocd 😂 What's wrong with us?